src/components/DataTable.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <div v-if="!refData">
+    <SkeletonTable />
+  </div>
+  <div v-else>
+    <div v-if="allowExport" class="flex flex-row w-full gap-2 py-2" style="justify-content: flex-end;">
+      <Button label="Export Selected" :disabled="refSelectedRows.length === 0" @click="exportCSV" />
+      <Button label="Export Whole Table" @click="exportFullCSV" />
+    </div>
+    <DataTable 
+      :value="refData" 
+      :rowHover="true" 
+      :reorderableColumns="true" 
+      :resizableColumns="true"
+      v-model:selection="refSelectedRows"
+      removableSort
+      sortMode="multiple"
+      showGridlines
+      v-model:filters="refFilters"
+      filterDisplay="menu"
+    >
+      <Column 
+        selectionMode="multiple" 
+        headerStyle="width: 3rem"
+      ></Column>
+      <Column 
+        v-for="col of columns" 
+        :key="col.field" 
+        :header="col.header" 
+        :field="col.field"
+        :dataType="col.dataType"
+        :sortable="(col.sort !== false)"
+        :showFilterMatchModes="!col.useMultiSelect"
+      >
+        <template v-if="col.dataType === 'date'" #body="{ data }">
+          {{ getFormattedDate(data[col.field]) }}
+        </template>
+        <template v-if="col.dataType" #filter="{ filterModel }">
+          <InputText 
+            v-if="col.dataType === 'text' && !col.useMultiSelect"
+            type="text" v-model="filterModel.value" 
+            class="p-column-filter" 
+            placeholder="Search" 
+          />
+          <MultiSelect 
+            v-if="col.useMultiSelect"
+            v-model="filterModel.value" 
+            :options="refOptions[col.field]" 
+            placeholder="Any"
+            :showToggleAll="false" 
+            class="p-column-filter" 
+          />
+          <Calendar 
+            v-if="col.dataType === 'date' && !col.useMultiSelect"
+            v-model="filterModel.value" 
+            dateFormat="mm/dd/yy" 
+            placeholder="mm/dd/yyyy" 
```

Comment:
```suggestion
            dateFormat="mm/dd/yyyy" 
            placeholder="mm/dd/yyyy" 
```

Should these two date formats match up?